code/game/objects/items/tools/shovel_tools.dm

```diff
@@ -143,25 +143,31 @@
 // Entrenching tool.
 /obj/item/tool/shovel/etool
 	name = "entrenching tool"
-	desc = "Used to dig holes and bash heads in. Folds in to fit in small spaces."
+	desc = "Used to dig holes and bash heads in. Folds in to fit in small spaces. Use a machete on it to sharpen it."
 	icon = 'icons/Marine/marine-items.dmi'
 	icon_state = "etool"
 	force = 30
 	throwforce = 2
 	item_state = "crowbar"
+	hitsound = "swing_hit"
 	w_class = WEIGHT_CLASS_BULKY //three for unfolded, 3 for folded. This should keep it outside backpacks until its folded, made it 3 because 2 lets you fit in pockets appearntly.
 	dirt_overlay = "etool_overlay"
 	dirt_amt_per_dig = 5
 	shovelspeed = 20
+	var/sharpened = FALSE
 
 
 /obj/item/tool/shovel/etool/update_icon()
 	if(folded) icon_state = "etool_c"
+	else if(sharpened) icon_state = "etool_s"
 	else icon_state = "etool"
```

Comment:
So this is pretty bad, but since it's already former behaviour i think tivi will leave you alone

Reply:
yeah i thought that too